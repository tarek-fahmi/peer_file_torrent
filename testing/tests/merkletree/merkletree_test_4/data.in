Completed Component - File w/ Incomplete Chunks
./testing/bin/pkg_main ./testing/resources/invalid_1.bpkg -min_hashes
./testing/bin/pkg_main ./testing/resources/invalid_2.bpkg -min_hashes
