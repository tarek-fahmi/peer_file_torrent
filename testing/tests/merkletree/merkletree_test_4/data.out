164af2435b45ae96e5f51c258f698a642c0b2927a246b15b975f117963d1c64d
40fd3d707f40e37bc4c6cfee574117ceeeb819980429a7a1073bba1c70b6f78c